canonical-web-and-design/ubuntu.com
# Remove the number from the distributions provided

## Done
Remove the number from the distributions provided

## QA
- Check the code matches the [copy doc](https://docs.google.com/document/d/1pLRWJZSpE04nYt-yHBtCjxGI7VZsigcP2UJIBmR8i-0/edit)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9196
